apps/rule-manager/client/src/ts/components/layout/Page.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { EuiPageTemplate, EuiProvider } from "@elastic/eui";
-import { Header } from "./Header";
+import {Header, headerHeight} from "./Header";
```

Comment:
I noticed when that my IDE did some automatic formatting, which made my PR a bit noisy (e.g. adding spaces around the imports). I thought about adding some prettier config to the project to enforce some consistent formatting, if that would be helpful?

Reply:
Will do now 👍